docs/getting_access/eessi_limactl.md

```diff
@@ -0,0 +1,170 @@
+# Installing EESSI with limactl on MacOS
+
+## Installation of Lima
+
+see lima documentation: [https://lima-vm.io/docs/installation/](https://lima-vm.io/docs/installation/)
+
+``` { .bash .copy }
+brew install lima
+```
+
+## Installing EESSI in limactl with EESSI template
+
+### Example `eessi.yaml` file
+
+Use the EESSI template to install a virtual machine with eessi installed. Create a `eessi.yaml` file
+
+=== "Install a virtual machine with a Debian image"
+    ``` { .bash .copy }
+    # A template to use the EESSI software stack (see https://eessi.io) on macOS
+    # $ limactl start ./eessi.yaml
+    # $ limactl shell eessi
+
+    images:
+    # Try to use release-yyyyMMdd image if available. Note that release-yyyyMMdd will be removed after several months.
+    - location: "https://cloud.debian.org/images/cloud/bookworm/20240429-1732/debian-12-genericcloud-amd64-20240429-1732.qcow2"
+      arch: "x86_64"
+      digest: "sha512:6cc752d71b390c7fea64b0b598225914a7f4adacd4a33fa366187fac01094648628e0681a109ae9320b9a79aba2832f33395fa13154dad636465b7d9cdbed599"
+    - location: "https://cloud.debian.org/images/cloud/bookworm/20240429-1732/debian-12-genericcloud-arm64-20240429-1732.qcow2"
+      arch: "aarch64"
+      digest: "sha512:59afc40ad0062ca100c9280a281256487348c8aa23b3e70c329a6d6f29b5343b628622e63e0b9b4fc3987dd691d5f3c657233186b3271878d5e0aa0b4d264b06"
+    # Fallback to the latest release image.
+    # Hint: run `limactl prune` to invalidate the cache
+    - location: "https://cloud.debian.org/images/cloud/bookworm/latest/debian-12-genericcloud-amd64.qcow2"
+      arch: "x86_64"
+    - location: "https://cloud.debian.org/images/cloud/bookworm/latest/debian-12-genericcloud-arm64.qcow2"
+      arch: "aarch64"
+
+    mounts:
+    - location: "~"
+    - location: "/tmp/lima"
+      writable: true
+    containerd:
+      system: false
+      user: false
+    provision:
+    - mode: system
+      script: |
+        #!/bin/bash
+        wget -P /tmp https://ecsft.cern.ch/dist/cvmfs/cvmfs-release/cvmfs-release-latest_all.deb
+        sudo dpkg -i /tmp/cvmfs-release-latest_all.deb
+        rm -f /tmp/cvmfs-release-latest_all.deb
+        sudo apt-get update
+        sudo apt-get install -y cvmfs
+        if [ ! -f /etc/cvmfs/default.local ]; then
+            sudo echo "CVMFS_HTTP_PROXY=DIRECT" >> /etc/cvmfs/default.local
+            sudo echo "CVMFS_QUOTA_LIMIT=10000" >> /etc/cvmfs/default.local
+        fi
+        sudo cvmfs_config setup
+    probes:
+    - script: |
+        #!/bin/bash
+        set -eux -o pipefail
+        if ! timeout 30s bash -c "until ls /cvmfs/software.eessi.io >/dev/null 2>&1; do sleep 3; done"; then
+          echo >&2 "EESSI repository is not available yet"
+          exit 1
+        fi
+      hint: See "/var/log/cloud-init-output.log" in the guest
+    ```
+=== "Install a virtual machine with an Ubuntu image"
```

Comment:
I think I would just use one example here (either Debian or Ubuntu) to keep this a bit cleaner/shorter. Or can you make the file contents foldable/expandable?

Reply:
Alternatively we could store the templates in a git repo and provide links to them?

Reply:
I have done it like on the native installation page, see https://www.eessi.io/docs/getting_access/native_installation/.
So only one shows up.

Reply:
Ah, okay, nice! :+1:

Reply:
In that case, I have another one (based on Rocky 9) 😄 
```
# A template to use the EESSI software stack (see https://eessi.io) on macOS
# $ limactl start ./eessi.yaml
# $ limactl shell eessi

images:
- location: "https://dl.rockylinux.org/pub/rocky/9.3/images/x86_64/Rocky-9-GenericCloud-Base-9.3-20231113.0.x86_64.qcow2"
  arch: "x86_64"
  digest: "sha256:7713278c37f29b0341b0a841ca3ec5c3724df86b4d97e7ee4a2a85def9b2e651"
- location: "https://dl.rockylinux.org/pub/rocky/9.3/images/aarch64/Rocky-9-GenericCloud-Base-9.3-20231113.0.aarch64.qcow2"
  arch: "aarch64"
  digest: "sha256:1948a5e00786dbf3230335339cf96491659e17444f5d00dabac0f095a7354cc1"
# Fallback to the latest release image.
# Hint: run `limactl prune` to invalidate the cache
- location: "https://dl.rockylinux.org/pub/rocky/9/images/x86_64/Rocky-9-GenericCloud.latest.x86_64.qcow2"
  arch: "x86_64"
- location: "https://dl.rockylinux.org/pub/rocky/9/images/aarch64/Rocky-9-GenericCloud.latest.aarch64.qcow2"
  arch: "aarch64"

mounts:
- location: "~"
- location: "/tmp/lima"
  writable: true
containerd:
  system: false
  user: false
provision:
- mode: system
  script: |
    #!/bin/bash
    sudo yum install -y https://ecsft.cern.ch/dist/cvmfs/cvmfs-release/cvmfs-release-latest.noarch.rpm
    sudo yum install -y cvmfs
    if [ ! -f /etc/cvmfs/default.local ]; then
        sudo echo "CVMFS_HTTP_PROXY=DIRECT" >> /etc/cvmfs/default.local
        sudo echo "CVMFS_QUOTA_LIMIT=10000" >> /etc/cvmfs/default.local
    fi
    sudo cvmfs_config setup
probes:
- script: |
    #!/bin/bash
    set -eux -o pipefail
    if ! timeout 30s bash -c "until ls /cvmfs/software.eessi.io >/dev/null 2>&1; do sleep 3; done"; then
      echo >&2 "EESSI repository is not available yet"
      exit 1
    fi
  hint: See "/var/log/cloud-init-output.log" in the guest
```

Reply:
Done